script/test.sh

```diff
@@ -12,6 +12,36 @@ function get_elb {
   echo $(aws elbv2 describe-load-balancers | jq --arg host "harmony-$HARMONY_ENVIRONMENT-frontend" '.LoadBalancers[] | select(.LoadBalancerName == $host) | .DNSName' | tr -d '"')
 }
 
+cd ..
```

Comment:
Again not part of this PR, but it's making me think we should consider pulling harmony-ci-cd to the open source side of the world and sticking these scripts in a more common location, given there are big pieces copy/pasted.

Reply:
HARMONY-761 now